pkg/mimedb/db.go

```diff
@@ -1508,10 +1588,18 @@ var DB = map[string]struct {
 		ContentType:  "image/jpeg",
 		Compressible: false,
 	},
+	"jpf": {
+		ContentType:  "image/jpx",
```

Comment:
should this not be image/jpf? https://tools.ietf.org/html/rfc3745 lists jpf as preferred format

Reply:
Just noticed this is generated by gen-db.go

Reply:
jpf is also for `image/jpx` i.e `image/jpx` is same as `image/jpf` - to fix proper content-types users have to set it manually. This package is a guess work at best, works for most common files.